spotbugs-exclude.xml

```diff
@@ -1,6 +1,7 @@
+<!--
 /*
- * Copyright 2014-2020 Groupon, Inc
- * Copyright 2014-2020 The Billing Project, LLC
+ * Copyright 2020-2023 Equinix, Inc
```

Comment:
I think it would be safe to remove the `Equinix` copyright - none of the work has happened at Equinix - but probably ok to leave it too.